internal/tui/styles/theme.go

```diff
@@ -601,9 +601,26 @@ func ApplyForegroundGrad(input string, color1, color2 color.Color) string {
 
 	ramp := blendColors(len(clusters), color1, color2)
 	for i, c := range ramp {
-		fmt.Fprint(&o, CurrentTheme().S().Base.Foreground(c).Render(clusters[i]))
+		style := t.S().Base.Foreground(c)
+		if bold {
+			style.Bold(true)
+		}
+		clusters[i] = style.Render(clusters[i])
 	}
+	return clusters
```

Comment:
```suggestion
	return strings.Join(clusters, "")
```

imho doesn't make sense to return a slice...